riscv-executor/src/lib.rs

```diff
@@ -1098,12 +1185,14 @@ impl<'a, 'b, F: FieldElement> Executor<'a, 'b, F> {
                     set_col!(Y_free_value, val);
                 }
 
+                main_op!(set_reg,);
```

Comment:
hm maybe a function would be nicer than a macro for this? The mandatory trailing comma looks kinda odd

Reply:
fixed the trailing comma